### PR TITLE
Complete database seed script and associated tests

### PR DESCRIPTION
**Pull request by ShaneSutro**
@vinayaachar - if you get a chance, would love your feedback on this script and tests

### CODE REVIEW

File: database/seedFunctions.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    'Gujarati',
+    'Polish',
+    'Persian',
+    'Kannada',
+    'Romanian',
+  ];
+  const numberOfLangsToUse = generateNumberWithinRange(0, languages.length);
+  for (let i = 0; i < numberOfLangsToUse; i++) {
+    const randomLanguageIndex = generateNumberWithinRange(0, languages.length);
+    chosenLanguages.push(languages.splice(randomLanguageIndex, 1)[0]);
```
Comment (vinayaachar) [inline database/seedFunctions.js]: Nice logic here but is removal of languages from the language list necessary? 

Would `chosenLanguages.push(languages[randomLanguageIndex])` suffice ?

File: database/seedFunctions.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+const generateRandomPercentage = () => (Math.floor(Math.random() * 100) / 100);
+
+const generateNumberWithinRange = (min, max) => (Math.floor(Math.random() * (max - min) + min));
+
+const generateFillerText = async (options) => {
+  let text;
+  if (options.paras) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&paras=${options.paras}&format=text`);
+  } else if (options.sentences) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&sentences=${options.sentences}&format=text`);
```
Comment (vinayaachar) [inline database/seedFunctions.js:13]: Nice touch with async/await , something we haven't learnt yet. 

Other option would be : Axios returns a `Promise` and use `then` block to return response in a `cb` to the calling function

File: database/seedFunctions.js (hunk 1 above)
Comment (ShaneSutro) [reply database/seedFunctions.js]: Hmm good point! I initially was thinking I didn't want multiples of the same language in the list, thus the removals. I suppose I could have also just selected a random index range, like from 0-3 or from 2-9 and sliced the whole thing, that would reduce the time complexity.

Comment (ShaneSutro) [reply database/seedFunctions.js]: Changed - modified based on suggestion but took a slightly different route. Still, reduced down to constant time.

File: database/seedFunctions.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+  }
+  return skills;
+};
+
+const generateRecords = async (numToGenerate) => {
+  const records = [];
+  for (let i = 1; i < numToGenerate + 1; i++) {
+    console.log(`Creating record ${i}`);
+    const item = {
+      course_id: i, // 1 - 100
```
Comment (vinayaachar) [inline database/seedFunctions.js:130]: What happens when you seeding multiple times ? Won't there be courses with same ID's always starting from 1-100

File: database/seedFunctions.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+      recent_views: Math.floor(Math.random() * 1000000), // Random number between 0 and 1 million
+      description: await generateFillerText({ paras: 4 }), // Bacon ipsum - 4 paragraphs
+      learner_career_outcomes: await generateLearnerCareerOutcomes(),
+      metadata: await generateMetadata(),
+      what_you_will_learn: await generateWhatYouWillLearn(),
+      skills_you_will_gain: await generateSkillsYouWillGain(),
+    };
```
Comment (vinayaachar) [inline database/seedFunctions.js:137]: Very nice use of all the async functions

**Pull request merged**